src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,30 @@
-import React from 'react';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+type Props = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => {
+  return (
+    <>
+      <div className="CommentInfo">
+        <div className="CommentInfo__title">
+          <h1
```

Comment:
h1 is the main title of the page and should be only one per page. 
Consider using some other tag for comments title